kiwi/storage/disk.py

```diff
@@ -481,9 +488,14 @@ def _add_to_map(self, name):
         partition_number = format(self.partitioner.get_id())
         if self.storage_provider.is_loop():
             device_base = os.path.basename(self.storage_provider.get_device())
-            device_node = ''.join(
-                ['/dev/', device_base, 'p', partition_number]
-            )
+            if self.partition_mapper == 'partx':
+                device_node = ''.join(
+                    ['/dev/', device_base, 'p', partition_number]
+                )
+            else:
+                device_node = ''.join(
+                    ['/dev/mapper/', device_base, 'p', partition_number]
+                )
```

Comment:
This condition should be flipped. Only when `kpartx` is used should we use `/dev/mapper` paths. That makes it obvious that `kpartx` is the special case.

Reply:
done